server/model/index.js

```diff
@@ -0,0 +1,15 @@
+const con = require('../database/index.js');
+
+let fetchAll = function(cb) {
```

Comment:
Are you attempting to fetch all of the reviews or fetch all of the reviews for a specific listing? If you are attempting to fetch all of the reviews for a specific listing, then you will need to refactor this query.

Reply:
@jcelder  query has been refactored to fetch reviews for a specific listing.